lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -1,532 +1,796 @@
 module Cerner
   module Resources
-
     R4_ENCOUNTER ||= {
-      "resourceType": "Encounter",
-      "id": "4027918",
+      "resourceType": 'Encounter',
```

Comment:
need to keep double quotes

Reply:
https://github.com/cerner/fhir.cerner.com/pull/415/commits/346d1c84ec066ff89fd8f5dfdb7fec58712cc176#diff-8915968a455f59526d7e4635365d9792R5